cmd/sg-public-access/sgpublicaccess_test.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/service/configservice"
+)
+
+type MockAWSConfigService struct {
+}
+
+func (m *MockAWSConfigService) PutEvaluations(input *configservice.PutEvaluationsInput) (*configservice.PutEvaluationsOutput, error) {
```

Comment:
can we move this to pkg/mocks/awsconfigservice.go and refer to this in both tools?